.gitignore

```diff
@@ -1,5 +1,5 @@
+*~
 project/project
-project/target
```

Comment:
not so important, but:
`*~` ignores text editor temp files such as `Client.scala~`
`project/target` is a sub case of `target/`